packages/babel-plugin-transform-runtime/README.md

```diff
@@ -33,3 +45,135 @@ require("babel-core").transform("code", {
   plugins: ["transform-runtime"]
 });
 ```
+
+## Technical details
+
+The `runtime` transformer plugin does three things:
+
+* Automatically requires `babel-runtime/regenerator` when you use generators/async functions.
+* Automatically requires `babel-runtime/core-js` and maps ES6 static methods and built-ins.
+* Removes the inline babel helpers and uses the module `babel-runtime/helpers` instead.
+
+What does this actually mean though? Basically, you can use built-ins such as `Promise`, `Set`, `Symbol` etc as well use all the Babel features that require a polyfill seamlessly, without global pollution, making it extremely suitable for libraries.
+
+### Regenerator aliasing
+
+Whenever you use a generator function or async function:
+
+```javascript
+function* foo() {
+
+}
+
+async function bar() {
+
+}
+```
+
+the following is generated:
+
+```javascript
+"use strict";
+
+var foo = regeneratorRuntime.mark(function foo() {
+  ...
+});
+
+function bar() {
+  return regeneratorRuntime.async(function bar$(context$1$0) {
+    ...
+  }, null, this);
+}
+```
+
+This isn't ideal as then you have to include the regenerator runtime which
+pollutes the global scope.
+
+Instead what the `runtime` transformer does it compile that to:
+
+```javascript
+"use strict";
+
+var _regeneratorRuntime = require("babel-runtime/regenerator");
+
+var foo = _regeneratorRuntime.mark(function foo() {
+  ...
+});
+
+function bar() {
+  return _regeneratorRuntime.async(function bar$(context$1$0) {
+    ...
+  }, null, this);
+}
+```
+
+This means that you can use the regenerator runtime without polluting your current environment.
+
+### `core-js` aliasing
+
+Sometimes you may want to use new built-ins such as `Map`, `Set`, `Promise` etc. Your only way
+to use these is usually to include a globally polluting polyfill.
+
+What the `runtime` transformer does is transform the following:
+
+```javascript
+var sym = Symbol();
+
+var promise = new Promise;
+
+console.log(arr[Symbol.iterator]());
+```
+
+into the following:
+
+```javascript
+"use strict";
+
+var _core = require("babel-runtime/core-js");
+
+var sym = _core.Symbol();
+
+var promise = new _core.Promise();
+
+console.log(_core.$for.getIterator(arr));
+```
```

Comment:
Like I said on my description (later edit) I mostly copied this from the old docs. I would also be ok with removing the technical details part and leave that to someone who might actually know what the details are and just leave the initial description, as right now there's nothing, and when I was trying to use this plugin it took me quite a bit of digging to understand what it was :(